<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStarDict::SettingsDialog</class>
 <widget class="QDialog" name="QStarDict::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QStarDict Settings</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <widget class="QWidget" name="tabGlobalSettings">
      <attribute name="title">
       <string>Global</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="instantSearchBox">
         <property name="text">
          <string>Instant search</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Pronounce words using this command:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="speechCmdEdit">
           <property name="toolTip">
            <string>Enter cmd for the speaching program.&lt;br&gt;If cmd contains &quot;%s&quot; it will be replaced to word, else word will be writen to stdin of speech process.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>369</width>
           <height>171</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDictionaries">
      <attribute name="title">
       <string>Dictionaries</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="dictsTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>112</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>32</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>24</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QToolButton" name="dictsMoveUpButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move up</string>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
           <property name="icon">
            <iconset resource="../plugins/stardict/pixmaps/pixmaps.qrc">
             <normaloff>:/icons/arrow-up.png</normaloff>:/icons/arrow-up.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dictsMoveDownButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move down</string>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
           <property name="icon">
            <iconset resource="../plugins/stardict/pixmaps/pixmaps.qrc">
             <normaloff>:/icons/arrow-down.png</normaloff>:/icons/arrow-down.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dictsShowInfoButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Show information about dictionary</string>
           </property>
           <property name="text">
            <string>Show info</string>
           </property>
           <property name="icon">
            <iconset resource="pixmaps/pixmaps.qrc">
             <normaloff>:/icons/dialog-information.png</normaloff>:/icons/dialog-information.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlugins">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="pluginsTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>112</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>32</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>24</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_3">
         <item>
          <widget class="QToolButton" name="pluginsShowInfoButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Show information about plugin</string>
           </property>
           <property name="text">
            <string>Info</string>
           </property>
           <property name="icon">
            <iconset resource="pixmaps/pixmaps.qrc">
             <normaloff>:/icons/dialog-information.png</normaloff>:/icons/dialog-information.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pluginsConfigureButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Configure plugin</string>
           </property>
           <property name="text">
            <string>Configure</string>
           </property>
           <property name="icon">
            <iconset resource="pixmaps/pixmaps.qrc">
             <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>112</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPopupWindowApperance">
      <attribute name="title">
       <string>Apperance</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Popup window apperance</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="2" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Opacity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="popupOpacitySpin">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Default width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="popupDefaultWidthSpin">
              <property name="minimum">
               <number>96</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>320</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Default height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="popupDefaultHeightSpin">
              <property name="minimum">
               <number>96</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>240</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>101</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPopupWindowBehavior">
      <attribute name="title">
       <string>Popup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="useScanBox">
         <property name="text">
          <string>Scan selection</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showIfNotFoundBox">
         <property name="text">
          <string>Show if word not found</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_7">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="useScanModifierBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show only if modifier pressed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modifierKeyBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Alt</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Control</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shift</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Win</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_6">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Timeout before hide after mouse over</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="timeoutBeforeHideSpin">
           <property name="specialValueText">
            <string>Don't hide</string>
           </property>
           <property name="suffix">
            <string> sec</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="pronounceWordBox">
           <property name="text">
            <string>Pronounce the word</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>123</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QStarDict::CSSEdit" name="apperanceCSSEdit" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QStarDict::CSSEdit</class>
   <extends>QWidget</extends>
   <header>cssedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>instantSearchBox</tabstop>
  <tabstop>speechCmdEdit</tabstop>
  <tabstop>dictsMoveUpButton</tabstop>
  <tabstop>dictsMoveDownButton</tabstop>
  <tabstop>dictsShowInfoButton</tabstop>
  <tabstop>popupDefaultWidthSpin</tabstop>
  <tabstop>popupDefaultHeightSpin</tabstop>
  <tabstop>popupOpacitySpin</tabstop>
 </tabstops>
 <resources>
  <include location="../plugins/stardict/pixmaps/pixmaps.qrc"/>
  <include location="pixmaps/pixmaps.qrc"/>
 </resources>
 <connections/>
</ui>
